script.tvmaze.scrobbler/libs/tvmaze_api.py

```diff
@@ -52,7 +52,8 @@ class ApiError(Exception):
     @staticmethod
     def extract_error_message_from_response(response):
         # type: (requests.Response) -> Text
-        if 'application/json' in response.headers.get('Content-Type', ''):
+        if 'application/json' in response.headers.get('Content-Type', '') \
```

Comment:
Please don't use the continuation symbol `\` but enclose the entire condition in brackets.